spacy/cli/apply.py

```diff
@@ -0,0 +1,141 @@
+import tqdm
+import srsly
+
+from itertools import chain
+from pathlib import Path
+from typing import Optional, List, Iterable, cast, Union
+
+from wasabi import msg
+
+from ._util import app, Arg, Opt, setup_gpu, import_code, walk_directory
+
+from ..tokens import Doc, DocBin
+from ..vocab import Vocab
+from ..util import ensure_path, load_model
+
+
+path_help = (
+    "Location of the documents to predict on. "
+    "Can be a single file in .spacy format or a "
+    ".jsonl file. Files with other extensions "
+    "are treated as single plain text documents. "
+    "If a directory is provided "
+    "it is traversed recursively to grab all files to "
+    "be processed. The files can be a mixture of .spacy, "
+    ".jsonl and text files. If .jsonl is provided the "
+    "specified field is going to be grabbed ('text' "
+    "by default)."
+)
```

Comment:
I know we do this sometime with error strings, but it would probably be easier to make this one big string with triple quotes?